frontend/src/metabase/querying/components/FilterValuePicker/SearchValuePicker/SearchValuePicker.tsx

```diff
@@ -2,21 +2,21 @@ import { useMemo, useState } from "react";
 import { useAsync, useDebounce } from "react-use";
 import { t } from "ttag";
 
-import { MultiSelect } from "metabase/ui";
+import { MultiAutocomplete } from "metabase/ui";
```

Comment:
The issue with dragging is back.

https://github.com/metabase/metabase/assets/6830683/df1b5829-ae35-4926-a58d-37f2ff29b355

cc: @npfitz who is working on something similar

Reply:
@kamilmielnik it's not back - the branch was not up-to-date with `master`. Now it is.